.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "editor.formatOnSave": true,
+    "python.formatting.provider": "black",
+    "python.linting.pylintEnabled": true,
+    "python.defaultInterpreterPath": "venv/Scripts/python.exe",
```

Comment:
does this work with a relative path or do you need something like `${workspaceFolder}`?

Reply:
It works, but I expect ` ${workspaceFolder}` will behave better in multi-root setups.

My main question was whether it'd be interpolated. The doc are cagey on that "The predefined variables are supported in a select number of setting keys in settings.json files such as...". Experimentation says yes.